consensus/babe/state.go

```diff
@@ -28,17 +28,9 @@ type BlockState interface {
 	SetBlock(*types.Block) error
 }
 
-// NetworkState interface for network state methods
-type NetworkState interface {
-	GetHealth() (*common.Health, error)
-	SetHealth(*common.Health) error
-	GetNetworkState() (*common.NetworkState, error)
-	SetNetworkState(*common.NetworkState) error
-	GetPeers() (*[]common.PeerInfo, error)
-	SetPeers(*[]common.PeerInfo) error
-}
-
```

Comment:
Why remove network state interface methods? Should we keep all state interfaces consistent?

Reply:
the state interfaces should only contain the functions that are used by that package. otherwise if we change something in state we have to change all the interfaces. also if it's unused it's kinda unnecessary